### ISSUE TITLE
Revise connector for GloBI resource

### ISSUE THREAD

**Issue by eliagbayani** [EOL/ContentImport#5]
Re: Revise connector for GloBI resource

From Jen (Jira Bibalex):  @jhammock  @eliagbayani 

This can wait until we're happy with the completeness of the publish including the pre-publish delete; I don't want to confuse the process. But before I lose track, I want to record this case here. I know our attribution format for GloBI records has always been unusual, partly because of the fields they use. I wonder if they made a change during the "supported by/refuted by" project that diverted some attribution away from our connector. Check out this EOL record:

https://eol.org/records/R20-PK112742945

I believe it corresponds to this GloBI search result:
https://www.globalbioticinteractions.org/?interactionType=interactsWith&sourceTaxon=Spirobranchus%20giganteus&targetTaxon=Bambusa

The GLoBI record "is supported by" a record that includes a specific provider occurrence record, which the EOL record doesn't have. I'd like to track that down and use it for the Source field, when available. If it's not always populated, we can use the current method as a backup.

**Comment by eliagbayani** [comment]
Hi Jen @jhammock ,
Sorry, I'm not sure if I've reported this already. Maybe in Jira or email or maybe not yet :-) 
And maybe they were just excluded by default.

Anyway, GloBI has these associations.
It does not exist in our EOL terms file.
If it does, it is not of type = 'association'.

http://purl.obolibrary.org/obo/RO_0002303   n=386
http://purl.obolibrary.org/obo/RO_0008509   n=722
http://purl.obolibrary.org/obo/RO_0002555   n=15
http://purl.obolibrary.org/obo/RO_0002236   n=1
http://purl.obolibrary.org/obo/RO_0002804   n=22,629
http://purl.obolibrary.org/obo/RO_0002807   n=1,915
http://purl.obolibrary.org/obo/RO_0002805   n=231

Thanks.

**Comment by jhammock** [comment]
I believe Katja reviewed and deliberately excluded some association terms, and I daresay some of the documentation of that is lost in the mists of JIRA. Some of these could also be new, certainly the smaller ones. They look valuable, but some of them look like they could benefit from taxon filters. It's not a ton of data. Even the ectomycorrhizal associations are only 33 plant species with 336 associates. @KatjaSchulz I had a quick look in the GloBI browser (except for habitat, which doesn't appear) and they look taxonomically reasonable, with a stray or two, probably mismapped names, but you may remember issues raised in a prior review. Do the terms ring any bells?

**Comment by KatjaSchulz** [comment]
I don't think we deliberately excluded any of these terms. I suspect that they have been added to GloBI fairly recently and that we just haven't caught up yet. We should probably add them to our terms file and then review the data quality to see if we need any filters to fix things.

**Comment by jhammock** [comment]
At second glance, co-roost is the one I found in the GloBI search (http://purl.obolibrary.org/obo/RO_0002801)  That's already in our terms file. 

http://purl.obolibrary.org/obo/RO_0008509 (has roost), like http://purl.obolibrary.org/obo/RO_0002303 (has habitat) doesn't appear in the GloBI search interface.

I don't know if this is the handiest option, but I plucked the relevant rows from GloBI's dwca-by-study export, attached here in case wanted.

[Archive.zip](https://github.com/user-attachments/files/31190371/Archive.zip)
